src/test/java/com/baeldung/web/controller/BookControllerFindByIsbnTest.java

```diff
@@ -48,7 +48,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 import com.baeldung.model.Book;
-import com.baeldung.web.error.EntityNotFoundException;
```

Comment:
EntityNotFoundException is wrongly imported, its a part of javax dependency, not a custom created by the project.